docs/swagger.yaml

```
@@ -134,4 +162,105 @@ paths:
+        "404":
+          description: Not Found
+        "500":
+          description: Internal Server Error
+      summary: Create new user
+      tags:
+      - user
+  /user/{uuid}:
+    delete:
+      description: get a user with uuid
+      parameters:
+      - description: user uuid
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)